Doc/sphinx/source/conf.py

```diff
@@ -0,0 +1,39 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'pyMaCh3'
+copyright = '2024, Ewan Miller'
```

Comment:
Hyper nitpicky but I think MaCh3 has a copyright licence 😆

Reply:
lol my plan to steal MaCh3 has been foiled